src/views/Admin/AdminUserDetail.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div v-if="user._id" class="user-detail">
+    <div class="user-detail__body">
+      <div>
+        <span
+          v-if="user.isAdmin"
+          class="user-detail__account-notice user-detail__account-notice--admin"
+          >Admin</span
+        >
+        <span
+          v-if="user.isBanned"
+          class="user-detail__account-notice user-detail__account-notice--ban"
+          >Banned</span
+        >
+        <span
+          v-if="user.isTestUser"
+          class="user-detail__account-notice user-detail__account-notice--test"
+          >Test account</span
+        >
+        <span
+          v-if="user.isFakeUser"
+          class="user-detail__account-notice user-detail__account-notice--fake"
+          >Fake account</span
+        >
+      </div>
+      <div class="user-detail__title">
+        {{ user.firstname }} {{ user.lastname }}
+      </div>
+      <div class="user-detail__subtitle">ID: {{ user._id }}</div>
+      <div class="user-detail__section">
+        <div class="user-detail__section-title">Joined</div>
+        <div>{{ createdAt }}</div>
+      </div>
+      <div class="user-detail__section">
+        <div class="user-detail__section-title">Email</div>
+        <div>{{ user.email }}</div>
+      </div>
+      <div v-if="partnerOrg" class="user-detail__section">
+        <div class="user-detail__section-title">Partner organization</div>
+        <div>{{ partnerOrg }}</div>
+      </div>
+      <div v-if="schoolName" class="user-detail__section">
+        <div class="user-detail__section-title">School</div>
+        <div>{{ schoolName }}</div>
+      </div>
+      <div v-if="user.zipCode" class="user-detail__section">
+        <div class="user-detail__section-title">Zip code</div>
+        <div>{{ user.zipCode }}</div>
+      </div>
+    </div>
+    <sessions-list :sessions="sessions" />
+  </div>
+</template>
+
+<script>
+import moment from "moment";
+import NetworkService from "@/services/NetworkService";
+import SessionsList from "@/components/Admin/SessionsList";
+
+const getUser = async userId => {
```

Comment:
Is there a benefit to have certain methods outside of `methods` in Vue?

Reply:
i think it gives methods a "private / utility" feel, like it's clearly not being attached to an @click event or anything. just a preference that felt right here -- i'm curious if vue best practices say anything about this